### PR TITLE
AO Menu

### PR DESCRIPTION
**Pull request by mjoyce91**
AO menu links that direct to existing Bureau pages

### CODE REVIEW

File: src/Components/BureauPage/BureauPage.jsx [hunk 1]
```diff
@@ -22,11 +22,11 @@ const BureauPage = () => {
...
-        <Route path="/profile/bureau/dashboard" render={() => <Dashboard {...dashboardProps} />} />
-        <Route path="/profile/bureau/stats" render={() => <Stats {...statsProps} />} />
-        <Route path="/profile/bureau/positionlists" render={() => <PositionLists {...posListsProps} />} />
-        <Route path="/profile/bureau/positionmanager/:type/:id" render={() => <PositionManagerDetails />} />
-        <Route path="/profile/bureau/positionmanager" render={() => <PositionManager />} />
+        <Route path="/profile/(bureau|ao)/dashboard" render={() => <Dashboard {...dashboardProps} />} />
+        <Route path="/profile/(bureau|ao)/stats" render={() => <Stats {...statsProps} />} />
+        <Route path="/profile/(bureau|ao)/positionlists" render={() => <PositionLists {...posListsProps} />} />
+        <Route path="/profile/(bureau|ao)/positionmanager/:type/:id" render={() => <PositionManagerDetails />} />
+        <Route path="/profile/(bureau|ao)/positionmanager" render={() => <PositionManager />} />
```
Comment (elizabeth-jimenez) [inline src/Components/BureauPage/BureauPage.jsx:29]: I didn't know we could do an `or` on these paths; very cool ⛄

File: src/Constants/Menu.js [hunk 2]
```diff
@@ -227,6 +227,38 @@ export const GET_PROFILE_MENU = () => MenuConfig([
...
+    text: 'AO',
+    route: '/profile/ao/positionmanager/',
+    icon: 'building-o',
+    toggleMenuSection: true,
+    expandedSection: true,
+    roles: [
+      'ao_user',
+      'superuser',
+    ],
+    children: [
```
Comment (elizabeth-jimenez) [inline src/Constants/Menu.js:240]: Didn't want the "I am the Bureau Stats" 😭  🏴‍☠️

File: src/Components/BureauPage/BureauPage.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/BureauPage/BureauPage.jsx:29]: That was new to me as well

**Review by elizabeth-jimenez — APPROVED** [review]
🍬 🍰 🥮 🍭 👍

**Pull request merged**